inference-engine/src/vpu/graph_transformer/src/middleend/passes/merge_eltwise_and_relu.cpp

```diff
@@ -28,15 +30,18 @@ class PassImpl final : public Pass {
 };
 
 void PassImpl::run(const Model& model) {
-    if (m_mode == MergeMode::DYNAMIC_NETWORK) {
-        VPU_PROFILE(mergeEltwiseAndReLUDynamic);
-        if (model->isStatic()) {
-            return;
-        }
-    } else if (m_mode == MergeMode::STATIC_NETWORK) {
-        VPU_PROFILE(mergeEltwiseAndReLUStatic);
-        if (model->isDynamic()) {
-            return;
+    const bool enableUnconditionalEltwiseReLUMergeForDynamicNetworks = CompileEnv::get().config.enableUnconditionalEltwiseReLUMerge;
```

Comment:
What does **Unconditional** actually mean? If we use conditional mode, then we decide to merge only in case when strides are supported also for dynamic case, right?
That is **Unconditional** mode related only to dynamic models?
Would it better to mention dynamism in the config key name?

Reply:
It's a good point, but when I tried it (MYRIAD_ENABLE_UNCONDITIONAL_ELTWISE_RELU_MERGE_FOR_DYNAMIC_NETWORKS) option name seemed to me too long. Originally, *unconditional* mode was related to all networks, but it turned out on some static networks it leads to performance degradation, so we made WA for static networks.

There is another point, that option is private, so if this options is used, it's assumed that user understand what he does anyway, so name detalization does not make as much sense as for public option.

@taka-no-me suggested `MYRIAD_FUSE_ELTWISE_RELU`, but it looks to me confusing because it looks like it turns off fusing for all cases. @Maxim-Doronin what do you think?

Please suggest specific option name if you have anything.

Once there is a variant all agree about I'll change the name, otherwise I suggest to leave it as is to integrate fix faster.

Reply:
Taking into consideration this 
> There is another point, that option is private, so if this options is used, it's assumed that user understand what he does anyway, so name detalization does not make as much sense as for public option.

I'm ok with the current name

My suggestion would be `MYRIAD_ENABLE_UNCONDITIONAL_DYNAMIC_ELTWISE_RELU_MERGE`. But, as I understand, there may be eltwises with static shapes in dynamic networks, so it is not a good proposal.